extensions/amp-form/amp-form.md

```diff
@@ -60,28 +60,34 @@ Example:
         </label>
         <input type="submit" value="Subscribe">
     </fieldset>
+    <div submit-success>
+        Subscription successful!
```

Comment:
This would probably error, you'd need to encapsulate the text with a `template` element.